src/test/unit/sanitize.test.js

```diff
@@ -132,6 +133,16 @@ describe('sanitizePII', function () {
     expect(sanitizedProfile.threads.length).toEqual(1);
   });
 
+  it('should sanitize counters if they are provided', function () {
+    const { originalProfile, sanitizedProfile } = setup({
+      shouldRemoveCounters: new Set([0]),
+    });
+
+    expect(ensureExists(originalProfile.counters).length).toEqual(1);
+    // The counter should be deleted now.
+    expect(ensureExists(sanitizedProfile.counters).length).toEqual(0);
+  });
+
```

Comment:
Can we maybe add a test case where the counters are _not_ sanitized when `shouldRemoveCounters` is not provided?